.github/workflows/github-ci.yml

```diff
@@ -25,6 +25,11 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
       # Runs a single command using the runners shell
       - name: Greetings and Init
         run: |
```

Comment:
Sice nerozumim te | ale log Hello World bych asi odstranil nebo zmenil za nejaky vypovidajici.  Greetings and Init take zustalo z nejakeho examplu?

Reply:
Hello world vymazáno ale ta fáze tam zůstat musí protože instaluje závislosti